api/modules/emailSubscriptions.js

```diff
@@ -31,15 +31,53 @@ const emailSubscriptions = module.exports = (() => {
                             text: 'New users email: ' + record.fields.Email,
                         },
                     ]
-                
+
                 })
-            )                
+            )
         })
         .then(sendgridResponse => {
             res.json({ message: 'Success' })
         })
     }
-    //method to insert Date in airtable 
+
+    const unsuscribeUser = (req, res) => {
+
+        airtable.updateRecord({
+            tableName: 'Subscriptions',
+            id: req.id,
+            fieldData: {
+                'Unsuscribed': true,
+            }
+        })
+        .then(record => {
+            return (
+                sendgrid.sendMessage({
+                    recipient: record.fields.Email,
+                    msg: [
+                        {
+                            to: record.fields.Email,
+                            from: 'contact@setlife.education',
+                            subject: 'Setlife Newsletter',
+                            text: 'The unsubscription will take effect within 3 days',
```

Comment:
Change text to `We have received your request to unsubscribe from SetLife emails and this has been automatically processed in our system. After this email we won't bother you again! If for some reason you continue to receive emails, we apologize in advance and ask that you please send a message to contact@setlife.network so we can resolve this manually. Thanks and have a great day!`